.github/actions/tag_trigger_action/action.yml

```diff
@@ -0,0 +1,26 @@
+name: tag_trigger_action
+description: Trigger a workflow on a tag addition
```

Comment:
I wonder if this should be called something like `job_execution_gate`? It's doing more than just checking a tag value.

Also, it's specifically setting the "run_e2e" output which seems too specific if we're making this a reusable action. Maybe it should just output "continue_execution=true|false"?

Reply:
if we use `run-e2e` test, the re-usable action is unnecessary